src/components/calculator.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="calc-input-group">
+        <input className="calc-input" type="text" value="0" readOnly />
+      </div>
+      <ul className="buttons-container">
+        <li className="button-row">
+          <button className="calc-btn" type="button">
+            AC
+          </button>
+          <button className="calc-btn" type="button">
+            +/-
+          </button>
+          <button className="calc-btn" type="button">
+            %
+          </button>
+          <button className="calc-btn btn-operation" type="button">
+            &#247;
+          </button>
+        </li>
+        <li className="button-row">
+          <button className="calc-btn" type="button">
+            7
+          </button>
+          <button className="calc-btn" type="button">
+            8
+          </button>
+          <button className="calc-btn" type="button">
+            9
+          </button>
+          <button className="calc-btn btn-operation" type="button">
+            x
+          </button>
+        </li>
+        <li className="button-row">
+          <button className="calc-btn" type="button">
+            4
+          </button>
+          <button className="calc-btn" type="button">
+            5
+          </button>
+          <button className="calc-btn" type="button">
+            6
+          </button>
+          <button className="calc-btn btn-operation" type="button">
+            -
+          </button>
+        </li>
+        <li className="button-row">
+          <button className="calc-btn" type="button">
+            3
+          </button>
+          <button className="calc-btn" type="button">
+            2
+          </button>
+          <button className="calc-btn" type="button">
+            1
+          </button>
+          <button className="calc-btn btn-operation" type="button">
+            +
+          </button>
+        </li>
+        <li className="button-row">
+          <button className="calc-btn btn-zero" type="button">
+            0
+          </button>
+          <button className="calc-btn" type="button">
+            .
+          </button>
+          <button className="calc-btn btn-operation" type="button">
+            =
+          </button>
+        </li>
+      </ul>
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
Nice job so far. Your project is looking great. However, as requested by the previous reviewer, kindly a create a js component and import it to be used here. You can remove all the button elements, put them in a different component, and import it here, just as you have done in the app.js file. Remember, the project requirements expect you to have at least one child component as shown below :point_down: 

![Screenshot from 2023-06-15 07-29-22](https://github.com/hagan199/Math-Magicians/assets/65233860/66046c82-9b4d-4c44-b5a8-d64e2f3e775a)